package.json

```diff
@@ -11,14 +11,21 @@
     "clean": "rimraf dist/",
     "build": "run-s lint clean babel",
     "babel": "babel src/ -d dist/",
-    "list:routes": "babel-node ./resources/scripts/list-routes.js"
+    "list:routes": "babel-node ./resources/scripts/list-routes.js",
+    "migrate:make": "knex migrate:make",
+    "migrate:latest": "knex migrate:latest",
```

Comment:
Let's make this script just `migrate`. So, that we can run it with just `npm run migrate`.